datahub-classify/setup.py

```diff
@@ -27,9 +29,12 @@ def get_long_description():
     "flake8>=3.8.3",
     "flake8-tidy-imports>=4.3.0",
     "isort>=5.7.0",
-    "mypy>=0.901,<0.920",
+    # Because of https://github.com/python/mypy/issues/13627 issue, set the mypy version to 0.981, we can revisit this in future
+    "mypy>==0.981",
```

Comment:
this doesn't look like a valid type spec

Reply:
Corrected the typo